src/fromager/packagesettings.py

```diff
@@ -142,6 +143,25 @@ def validate_destination_filename(cls, v):
         return v
 
 
+class BuildOptions(pydantic.BaseModel):
+    """Build system options"""
+
+    model_config = MODEL_CONFIG
+
+    cpu_scaling: int = 1
+    """scale parallel jobs by available CPU cores
+
+    1: as many parallel jobs as CPU cores
+    2: allocate 2 cores per job
+    """
+
+    memory_scaling: int = 2
```

Comment:
Similarly here, expressing this as a unit, rather than just a scaling factor, would make it easier to understand how to adjust the value. Is it counting 512MB at a time?

Reply:
I have renamed the option to ` memory_per_job_gb`. It's now a floating point number `>=0.1`. `0.5` is 512 MB.